mne/datasets/utils.py

```diff
@@ -252,7 +252,7 @@ def _data_path(path=None, force_update=False, update_path=True, download=True,
              'datasets/foo.tgz',
         misc='https://codeload.github.com/mne-tools/mne-misc-data/'
              'tar.gz/%s' % releases['misc'],
-        sample="https://osf.io/86qa2/download",
+        sample="https://osf.io/j4ms3/download",
```

Comment:
Instead of doing this, we should add the `mne.datasets.fetch_fsaverage_3layer_bem` function. We want it anyway so people don't need to download 1.5GB+ of data for this functionality, and the recent dataset update is problematic.